libcxxabi/test/test_demangle.pass.cpp

```diff
@@ -30114,6 +30114,9 @@ const char* cases[][2] =
      " std::allocator<char>>::basic_string()"},
     {"_ZN1SB8ctor_tagC2Ev", "S[abi:ctor_tag]::S()"},
     {"_ZN1SB8ctor_tagD2Ev", "S[abi:ctor_tag]::~S()"},
+
+    // clang builtin type transform
+    {"_Z2f5IiEvu7__decayIT_E", "void f5<int>(__decay(int))"},
```

Comment:
Should we add tests to cover the other type transforms from: https://github.com/llvm/llvm-project/commit/e9ef45635b77598fc9ce0cd38d7d3f8c9d88a49d